<launch>
  <param name="robot_description" command="cat $(find hebiros_description)/urdf/x_demo.urdf"/>

  <node name="hebiros_node" pkg="hebiros" type="hebiros_node" respawn="false" output="screen"/>

  <node name="example_moveit_node" pkg="hebiros_advanced_examples" type="example_moveit_node"/>

  <include file="$(find x_demo)/launch/demo.launch"/>

</launch>
